block/manager_test.go

```diff
@@ -57,26 +56,26 @@ func WithinDuration(t *testing.T, expected, actual, tolerance time.Duration) boo
 func getManager(t *testing.T, backend goDA.DA) *Manager {
 	logger := test.NewLogger(t)
 	return &Manager{
-		dalc:       da.NewDAClient(backend, -1, -1, nil, logger),
-		blockCache: NewBlockCache(),
-		logger:     logger,
+		dalc:        da.NewDAClient(backend, -1, -1, nil, logger),
+		headerCache: NewHeaderCache(),
+		logger:      logger,
 	}
 }
 
 // getBlockBiggerThan generates a block with the given height bigger than the specified limit.
-func getBlockBiggerThan(blockHeight, limit uint64) (*types.Block, error) {
-	for numTxs := 0; ; numTxs += 100 {
-		block := types.GetRandomBlock(blockHeight, numTxs)
-		blob, err := block.MarshalBinary()
-		if err != nil {
-			return nil, err
-		}
-
-		if uint64(len(blob)) > limit {
-			return block, nil
-		}
-	}
-}
+// func getBlockBiggerThan(blockHeight, limit uint64) (*types.SignedHeader, *types.Data, error) {
+// 	for numTxs := 0; ; numTxs += 100 {
+// 		header, data := types.GetRandomBlock(blockHeight, numTxs)
+// 		blob, err := header.MarshalBinary()
+// 		if err != nil {
+// 			return nil, nil, err
+// 		}
+
+// 		if uint64(len(blob)) > limit {
+// 			return header, data, nil
+// 		}
+// 	}
+// }
```

Comment:
Is this needed? Can it just be deleted?

Reply:
i will adapt this as part of fixing: https://github.com/rollkit/rollkit/issues/1808